MetaCocktailsSwiftData/Model/Cocktails/Death and Co/Alabaster.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Alabaster.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James Menkal on 8/17/24.
+//
+
+import Foundation
+
+var alabaster = Cocktail(cocktailName: "Alabaster",
+                         glasswareType: .nickAndNora,
+                         author:Author(person: AuthorNames.jakePowell.rawValue,
+                                       place: AuthorPlaces.deathAndCo.rawValue,
+                                       year: "Denver, 2022"),
```

Comment:
Just the year is fine here. We can lump all death and do recipes into the same place, the author will differentiate them